src/ui/reducers/app.ts

```diff
@@ -10,7 +10,7 @@ function initialAppState(): AppState {
     unexpectedError: null,
     theme: "theme-light",
     // Whether or not the developer tools toolbox is opened.
-    isToolboxOpen: prefs.isToolboxOpen,
+    isToolboxOpen: false,
```

Comment:
hmm, i think it is nice to use the pref here. In general, i'm a fan of persisting our UI settings

Reply:
Gotcha!